src/webui/src/components/Sessionpro.tsx

```diff
@@ -88,6 +88,20 @@ class Sessionpro extends React.Component<{}, SessionState> {
         };
     }
 
+    convertTime = (num: number) => {
+        let hour = Math.floor(num / 3600 % 24);
```

Comment:
why mod 24 here?  it seems there is no day unit here.

Reply:
ok